spec/integration/patches/railtie_spec.rb

```diff
@@ -5,12 +5,6 @@
     cmd.stdout
   end
 
-  context "console" do
-    it "loads the console config into the selected console" do
-      expect(cmd_stdout).to include('sandbox')
```

Comment:
Not familiar here, but is there some other integration test that should be added in place of this?